examples/nginx-app/with-pv.yaml

```diff
@@ -43,7 +43,7 @@ metadata:
   name: nginx-deployment
   namespace: nginx-example
 spec:
-  replicas: 2
+  replicas: 1
```

Comment:
Ah, interesting! I think we'll probably go ahead and remove the node labeling bit from that doc, but let's wait for @abiogenesis-now to chime in before doing so.